### ISSUE TITLE
download_backtest_data.py should fetch pairlist from config

### ISSUE THREAD

**Issue by hroff-1902** [freqtrade/freqtrade#1857]
... If it runs with config (or even if it runs with default config.json).

Exactly as freqtrade hyperopt and backtesting commands implement it.

The pairlist file should be read and used only if specified explicitely by the user in the command line if she needs to fetch a different (wider than in config) list of pairs or if the dynamic pairlist is set in her config.

**Comment by hroff-1902** [comment]
@xmatthias 

TBD, there are 2 options for this script:
* read default config if no --config is specified. In this case, in order to run without config at all (with options in the command line only), --no-config option should be introduced.
* do not read default config, I.e., the script is either run with config explicitely specified in the --config option (options) or run without config with all data specified in other command line options.

Which one to implement?

**Comment by xmatthias** [comment]
personally, i would prefer the 2nd option as that's (kinda) how it works now, and will still allow for a seperate pair-file.

I think the only thing that needs changing is to use the pair-list from the configuration if a config is provided, and no pairlist file is provided.

**Comment by hroff-1902** [comment]
> as that's (kinda) how it works now

Well, I broke it couple of months ago (when multiple configs were introduced). 😎  No tests for scripts, so this left unnoticed...

Currently the script reads the default config if not specified explicitely. Fixed it, commit will follow.

I also added support for -v and --logfile options -- for same reasons why they exist in the app itself, maybe useful for devs (and for users).
